src/utils/shared/k8s/delete.go

```
@@ -110,6 +118,32 @@ func (o *ObjectDeleter) DeleteNamespace() error {
 	return err
 }
 
+// getAggregatedGroupVersions returns the set of group/versions that are served
+// by an aggregated APIService (spec.service is non-nil). Resources in those
+// groups are skipped during cluster-wide deletion sweeps because aggregated
+// servers frequently advertise the delete verb on read-only virtual resources
+// and fail the call with "operation not supported".
+func (o *ObjectDeleter) getAggregatedGroupVersions() (sets.String, error) {
```

golangci-lint (2.12.1)

[error] SA1019: sets.String is deprecated: use generic Set instead. new ways: s1 := Set[string]{} s2 := New[string]() 

(staticcheck)